src/database/UIDatabase.js

```diff
@@ -13,10 +13,14 @@ export class UIDatabase {
       case 'Supplier':
         results = results.filtered('isVisible == true AND isSupplier == true');
         break;
+      case 'mSupplyStore':
```

Comment:
Just reading around...it's less than unanimously agreed upon. Up to your preference